http-server/src/main/java/org/triplea/server/remote/actions/RemoteActionsWebSocket.java

```diff
@@ -29,11 +27,6 @@ private RemoteActionsEventQueue getEventQueue(final Session session) {
         (RemoteActionsEventQueue) session.getUserProperties().get(ACTIONS_QUEUE_KEY));
   }
 
-  @OnClose
-  public void close(final Session session, final CloseReason closeReason) {
-    getEventQueue(session).removeSession(session);
-  }
-
   /**
```

Comment:
Also mentioned in #5802
This class and GameListingWebsocket are almost identical

Reply:
Yes, I was looking for a way to share the annotations and pass an implementation, but the annotation scanner does not pick up annotated methods from a parent class. We could consider merging all websockets together into just one based on connection source. If that seems like a good idea, we can  continue that conversation in an issue and follow-up.

Reply:
Yeah, I have no problem with that